### DISCUSSION TITLE
New release with proto 3 support?

### DISCUSSION THREAD

**Discussion by nacho** [protobuf-c/protobuf-c#636] [Q&A]
Hey folks,

I wanted to check if there is any ETA to make a new release with proto 3 support since the patches are already merged in next.

**Comment by edmonds** [comment]
Probably pretty soon.

**Comment by ringerc** [comment]
@edmonds I haven't been able to find clear release notes saying which version adds/added proto3 support. Is it 1.2.0? 1.3.0? The coming one? Changelogs (https://github.com/protobuf-c/protobuf-c/releases/) are empty.

Edit: grepping sources suggest 1.3.0 includes proto3, 1.2.0 does not.
